.eslintrc.js

```diff
@@ -47,11 +64,12 @@ module.exports = {
         'CONDITIONALS',
         'RENDER_MODIFIERS',
         'UNIQUE',
-        'BINDING',
+        'TWO_WAY_BINDING',
+        'OTHER_DIRECTIVES',
         'EVENTS',
-        'CONTENT',
         'GLOBAL',
-        'OTHER_ATTR'
+        'OTHER_ATTR',
+        'CONTENT'
```

Comment:
Should we consider bumping CONTENT up ? I would expect v-html to be defined as one of the top ones.... 🤔